### PR TITLE
fix: Replace Che links with DevSpaces ones

### PR DESCRIPTION
**Pull request by tolusha**
### Reference issue
https://issues.redhat.com/browse/CRW-6352

### CODE REVIEW

File: devspaces-operator-bundle/build/scripts/sync-che-olm.sh [hunk 1]
```diff
@@ -234,6 +234,38 @@ for CSVFILE in ${TARGETDIR}/manifests/devspaces.csv.yaml; do
...
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-2-for-the-bitbucket-cloud/
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-1-for-a-bitbucket-server/
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-2-for-microsoft-azure-devops-services
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-2-for-github/
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-2-for-gitlab/
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-2-for-the-bitbucket-cloud/
+    https://www.eclipse.org/che/docs/stable/administration-guide/configuring-oauth-1-for-a-bitbucket-server/
+  )
+  DEVSPACES_LINKS=(
+    https://access.redhat.com/documentation/en-us/red_hat_openshift_dev_spaces/3.13/html/administration_guide/configuring-devspaces
```
Comment (dmytro-ndp) [inline devspaces-operator-bundle/build/scripts/sync-che-olm.sh:252]: version number 3.13 is hardcoded, and will be outdate after the next release of Dev Spaces 3.14

Do you think it's possible to make the Dev Spaces version flexible in the links to documentation, e.g. by re-using script https://github.com/redhat-developer/devspaces-images/blob/407b67226e0afb506c8f7aee81cb9de03aef7e1e/devspaces-code/branding/branding.sh in here?

Comment (tolusha) [reply devspaces-operator-bundle/build/scripts/sync-che-olm.sh]: Thank you for noticing hardcoded version. I've replaced it with env var.

**Comment by dmytro-ndp** [comment]
@tolusha: PR looks good to merge commonly.

Cosmetic notice: you could probably consider usage of https://docs.redhat.com/en/documentation/red_hat_openshift_dev_spaces/ links, taking into account the planning transition of product documentation, announced in the article https://source.redhat.com/groups/public/t3/technical_topic_torrent_blog/transition_of_red_hat_documentation_to_docsredhatcom

**Comment by tolusha** [comment]
@dmytro-ndp 
Sure, I will

**Pull request merged**